tests/tests_unit/test_api/test_files.py

```diff
@@ -170,12 +188,20 @@ def test_create_with_label_request(self, mock_file_create_response):
         assert FileMetadata._load(response_body) == returned_file_metadata
         assert all(body["labels"][0]["externalId"] == "WELL LOG" for body in [request_body, response_body])
 
-    def test_create(self, mock_file_create_response):
-        file_metadata = FileMetadata(name="bla")
+    def test_create_with_geoLocation(self, mock_file_create_response, mock_geo_location):
+        file_metadata = FileMetadata(name="bla", geo_location=mock_geo_location)
         returned_file_metadata, upload_url = FILES_API.create(file_metadata)
         response_body = mock_file_create_response.calls[0].response.json()
         assert FileMetadata._load(response_body) == returned_file_metadata
-        assert response_body["uploadUrl"] == upload_url
```

Comment:
Might as well keep this assertion.

Reply:
Copied it from the `new_file_with_label()` fixture below, but I believe it's because the file is not always available immediately after upload.